resources/lang/en/thoughts/alive/general.json

```
@@ -6068,7 +6079,25 @@
             "Is comforted by r_c after a flashback",
             "Doesn't seem to enjoy hunting anymore",
             "Is avoiding {PRONOUN/m_c/poss} Clanmates",
-            "Is startled after r_c snuck up behind {PRONOUN/m_c/object}"
+            "Is startled after r_c snuck up behind {PRONOUN/m_c/object}",
+			"Is worried about having a flashback during a patrol",
+			"Is desperately trying to not make a noise despite panicking",
+			"Is attempting to do breathing exercises on {PRONOUN/m_c/inposs} own",
+			"Is purring to {PRONOUN/m_c/self} to try and calm down after a flashback",
+			"Is feeling relieved to have gone a moon without a flashback",
+			"Is proud of {PRONOUN/m_c/self} for calming down on {PRONOUN/m_c/poss} own",
+			"Doesn't seem to want to go near a certain area in the territory",
+			"Is keeping {PRONOUN/m_c/self} very occupied",
+			"Freezes up after hearing a familiar noise",
+			"Apologises to r_c for leaving abruptly",
+			"Is unsure what is causing {PRONOUN/m_c/object} to feel on edge",
+			"Is asking r_c what happened before {PRONOUN/m_c/subject} had a flashback",
+			"Unsheathes {PRONOUN/m_c/poss} claws when r_c asks what {PRONOUN/m_c/subject} saw in a flashback",
+			"Checks the camp entrance once, twice, thrice... just to be safe",
+			"Is almost sent into a panic at r_c scaring the life out of {PRONOUN/m_c/object}",
```

```suggestion
			"Is almost sent into a panic when r_c startles {PRONOUN/m_c/object}",
```
original is a bit clunky-sounding